### DISCUSSION TITLE
Backup to Android 12

### DISCUSSION THREAD

**Discussion by Z0Z0ne** [seedvault-app/seedvault#426] [Q&A] [answered]
Hi, my sister is trying to find a way to restore a backup made in CalyxOS onto Android 12 (which was installed on the same device, replacing CalyxOS). Is it possible to install Seedvault on a device that doesn't have it built-in?

**Comment by chirayudesai** [comment, marked as answer]
Unfortunately not easily, it has to be included in the OS.

Since it is able to access the data of all installed apps, the system wants to make sure rogue apps can't do that and thus only apps included with the OS are able to handle this.

**Reply by Z0Z0ne** [reply]
Thank you for the quick reply. 
That makes sense. Is there any way I can get any app data and download it to the corresponding app? 
Sorry if that is not possible and that sounds silly, I am not very experienced in in-depth stuff like this.
Thank you again for any help you can give me and the help you have already given me.

**Reply by chirayudesai** [reply]
Sorry to the bearer of bad news but there's no easy way to do that.
